vdb/__init__.py

```diff
@@ -1150,12 +1158,34 @@ def do_stepi(self, line):
 
 
                 # execute the instruction
-                if op.iflags & envi.IF_CALL and stepover:
-                    bp = vtrace.breakpoints.OneTimeBreak(op.va + op.size)
-                    self.trace.addBreakpoint(bp)
-                    self.trace.run()
+                if op.iflags & envi.IF_CALL and (stepover or module):
```

Comment:
There's a lot of overlap here with just a standard step over. So considering you have to know you're calling into a new map for this to really have value, why the split between this and step over?

Reply:
this is basically step-into unless it's about to call to a different loaded library, then it's step-over.  it allows for more context than stepo, but also avoids digging into the guts of KERNEL32 or LIBC.

Reply:
Still feels very duplicative, but I can see the use. I would like to know if we can detect if we step over a different memory map (not just a different library)  and display that. If it's a lot of work, you can skip it. It's just a nice to have imo

Reply:
we can make it do same-memory-map.  in fact, it's probably easier to write that code with confidence.  but harder paradigm to communicate and wrap your head around.

Reply:
We can keep it as is if it's conceptually easier. But we should figure out how to test these commands at some point.

Reply:
:+1: